react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-import chai, { expect } from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 import React from 'react';
-import {
-  Simulate,
-  renderIntoDocument,
-  scryRenderedDOMComponentsWithClass,
-  findRenderedDOMComponentWithClass
-} from 'react-dom/test-utils';
-import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
-import { findAllWithClass } from 'react-shallow-testutils';
-import Checkbox from './Checkbox';
-
-chai.use(sinonChai);
-
-const eventMatcher = sinon.match.instanceOf(SyntheticEvent);
+import { shallow } from 'enzyme';
```

Comment:
It's rendered as <CheckMarkIcon>, just not as a `<div>`with all the things inside. It's responsibility of CheckMarkIcon snapshots to capture it's guts